.gitignore

```diff
@@ -1 +1,2 @@
 dist/
+.idea
```

Comment:
I'd put this in your global ~/.gitignore -- I don't want to pollute our .gitignore with editor-specific configurations.